src/zoning-district/zoning-district.repository.schema.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from "zod";
+
+export const findByUuidRepoSchema = z.object({
+  id: z.string().uuid(),
+  label: z.string(),
+});
```

Comment:
The schema itself looks good but we'll need to reuse it across several endpoints. For example, there is actually a tax lot endpoint that returns zoning districts (#155). For this, we can move `z.object` portion to `schema/zoning-district` and define a `zoningDistrictEntitySchema`. We can then reuse the schema for the Repo Schemas. In this case:

**Note: suggestion if for illustrative purposes. Please validate in a full code editor**
```suggestion
export const findByUuidRepoSchema = zoningDistrictEntitySchema;
```

[Tax lot entity schema example](https://github.com/NYCPlanning/ae-zoning-api/blob/886f505433ddf9ac64987a0a9b46ae13f6d54e93/src/schema/tax-lot.ts#L36)

(As a general pattern, the spatial columns are left out of the Entity Schema. If they are needed, we can [extend](https://zod.dev/?id=extend) the base EntitySchema)

Reply:
Good call out, I've updated `src/schema/zoning-district.ts` to include the barebones entity schema so it can be expanded upon with more endpoints and used by other domains.